<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraCell</class>
 <widget class="QWidget" name="CameraCell">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="camOptionsComboBox"/>
     </item>
     <item>
      <widget class="QVideoWidget" name="camFeedWidget" native="true"/>
     </item>
     <item>
      <widget class="QLabel" name="poseOutput">
       <property name="text">
        <string>output for pose result</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeButton">
       <property name="text">
        <string>remove cam</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
